kustomize/base/nginx/templates/routes/openai.conf.template

```diff
@@ -1,10 +1,11 @@
+set $openai_upstream ${NGINX_UPSTREAM_NIM_LLM};
+
 location /openai {
 
    location ~* ^\/openai\/v1\/((engines\/.+\/)?(?:chat\/completions|completions|edits|moderations|answers|embeddings))$ {
       rewrite ^\/openai(\/.*)$ $1 break;
-      proxy_pass https://api.openai.com;
+      proxy_pass $openai_upstream;
```

Comment:
Use the same pattern used in the `nvidia.conf.template` i.e. `${NGINX_UPSTREAM_OPENAI}`